src/moditect/module-info.java

```diff
@@ -0,0 +1,11 @@
+module com.fasterxml.jackson.core {
+	exports com.fasterxml.jackson.core;
+
+	exports com.fasterxml.jackson.core.type to com.fasterxml.jackson.databind, com.jwebmp.core, com.fasterxml.jackson.datatype.jsr310;
+	exports com.fasterxml.jackson.core.io to com.fasterxml.jackson.databind, com.fasterxml.jackson.module.afterburner, com.fasterxml.jackson.datatype.jdk8;
```

Comment:
Also should be open to all: widely enough used, esp. something like `JsonEOFException`. Going forward could be refactored (for 3.0) to divide into public and internal types, but for now I think needs to be public.